doc/exts/pylint_messages.py

```diff
@@ -39,6 +42,8 @@ class MessageData(NamedTuple):
     bad_code: str
     details: str
     related_links: str
+    checker_module_name: str
```

Comment:
Is there a difference between this and `checker`, defined on L37?

Reply:
checker is the name of checker. Checker_module_name is full name of checker module e.g. pylint.extensions.bad_builtin

Reply:
Ah of course. Makes sense!